### DISCUSSION TITLE
醫療 AI 應用-心電圖分析📉

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#123] [General]
**新聞內容：**
長佳智能深耕數位醫療領域，將人工智慧應用於**醫學影像、輔助判讀、基因檢測結果分析、藥物研發**等，目前已取得 4 張 TFDA、1 張美國 FDA 醫材許可證，11/16日宣布，自主開發的心電圖分析軟體通過衛福部食藥署 (TFDA) 核准，取得智慧醫材許可證。
與中國醫藥大學附設醫院合作開發**心電圖分析軟體**，透過自主研發設計的神經網路架構技術，可辨識 13 種心律特徵，包括心房顫動、完全房室傳導阻滯、房性異位節律、竇性心律過快、ST 波段上升型心肌梗塞等。
即時判讀心電圖並達到 0.97 的準確率，可自動快速辨識心率異常、提升急診急性心肌梗塞病人，可及早發現並接受心導管手術治療，縮短到院至灌流時間。
長佳智能共有三大事業體，分別為醫療大數據、醫療 AI 軟體醫材與雲端平台，其中，醫療大數據已接獲許多國際藥廠訂單，醫療 AI 軟體醫材則向台灣與美國申請藥證中，雲端平台則是與國內多數人壽壽險公司合作，預計明年上半年將會陸續亮相。

**新聞心得：**
心臟對於人體來說非常重要，能及時偵測結果對於病情能大幅改善，而他們合作開發的分析軟體不僅具有高準確率，系統也能很快的完成判斷，有上該公司官網尋找其他服務，發現還有顱內出血、心臟檢查及兒童骨齡…的偵測，而且導入系統前後的處理時間真的差好多，原本需要好幾小時的事情，現在都能在短時間內達成，好令人佩服。

**新聞來源：**
https://news.cnyes.com/news/id/4772293?exp=a

![BA_report_normal](https://user-images.githubusercontent.com/90883526/142712807-c116ae2c-3c52-42b7-8b18-2bfa57d6d387.jpg)

![ECG_Atrial-Premature-Beat](https://user-images.githubusercontent.com/90883526/142712810-69e0971e-da94-4f9d-a498-06fe47243468.png)

**Comment by jim11261314** [comment]
這個新聞給我最大的感想是
原來醫療AI技術也算是醫材一種
台灣因為資金問題可能無法投入藥廠的建立
那麼利用資訊背景的建立醫療AI技術，或許是台灣可以接入的點
